public/src/components/channelManagement/bannerTests/bannerVariantPreview.tsx

```diff
@@ -114,16 +114,9 @@ const buildProps = (
   };
 };
 
-const bannerModules = {
-  DesignableBanner: {
-    path: 'designableBanner/DesignableBanner.js',
-    name: 'DesignableBanner',
-  },
-};
-
 const useStyles = makeStyles(({ palette, spacing }: Theme) => ({
   drawer: {
-    height: 'auto',
+    height: '400px',
```

Comment:
the downside to using an iframe is that we have to take a guess at the height. It will be scrollable within the iframe if the banner is taller than this